Xamarin.Essentials/Browser/Browser.android.cs

```diff
@@ -21,8 +21,10 @@ static Task PlatformOpenAsync(Uri uri, BrowserLaunchType launchType)
             {
                 case BrowserLaunchType.SystemPreferred:
                     var tabsBuilder = new CustomTabsIntent.Builder();
-                    var tabsIntent = tabsBuilder.Build();
                     tabsBuilder.SetShowTitle(true);
+                    var tabsIntent = tabsBuilder.Build();
+                    tabsIntent.Intent.SetFlags(ActivityFlags.ClearTop);
+                    tabsIntent.Intent.SetFlags(ActivityFlags.NewTask);
```

Comment:
Added the flags - also moved the `SetShowTitle` to _before_ the intent is built.